pyanaconda/core/configuration/anaconda.py

```diff
@@ -179,6 +179,16 @@ def can_configure_network(self):
         """Can we configure the network?"""
         return self._is_boot_iso or self._is_live_os
 
+    @property
+    def can_write_network_config(self):
```

Comment:
Could the rule be called `provides_network_config`? It seems to be similar to for example `provides_resolver_config`. The current name suggests that we can write to the network configuration of the installation system and that is not true.